examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e7f08ea5-2007-4c35-aae2-ed38eded5e16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import devito as dv\n",
+    "import sympy as sp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f4308415-d7ad-441c-b889-60ab4ce7155f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = dv.Grid(shape=(201, 201), extent=(1000., 1000.))\n",
+    "p = dv.TimeFunction(name='p', grid=grid, space_order=16)\n",
+    "v = dv.VectorTimeFunction(name='v', grid=grid, space_order=16, staggered=(None, None))\n",
+    "vx, vy = v\n",
+    "\n",
+    "# Material parameters\n",
+    "c = dv.Function(name='c', grid=grid, space_order=4)\n",
+    "rho = dv.Function(name='rho', grid=grid, space_order=4)\n",
+    "c.data[:] = 1.\n",
+    "rho.data_with_halo[:] = 1. # Encountering zeros in the halo here is bad\n",
+    "\n",
+    "# Define bouyancy for shorthand\n",
+    "b = 1/rho\n",
+    "# Define square celerity for shorthand\n",
+    "c2 = c**2  # FIXME: This prints missing the ^2 (string does contain **2 however)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6aecbe0e-daac-4235-b59c-d4a098bdbb84",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dt = grid.stepping_dim.spacing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8e44ac5a-0b68-40bc-8f3f-a7d30fb4964a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For setting up the 2nd-order ADER scheme\n",
+    "\n",
+    "# First time derivatives\n",
+    "pdt = rho*c2*dv.div(v)\n",
+    "vdt = b*dv.grad(p)\n",
```

Comment:
This is quite surprising, it's missing the time leap-frogging

```
p.forward = f(v, p))
v.forward=f(p.forward, v)
```

Reply:
Yeah, you don't need time leapfrogging for ADER. You also don't need any staggering to solve first-order systems of equations